.github/workflows/lighthouse.yml

```diff
@@ -0,0 +1,36 @@
+name: Lighthouse
+
+on: [pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v1
+      - name: Use Node.js 12.x
+        uses: actions/setup-node@v1
+        with:
+          node-version: 12.x
+      # Build needs around 340 seconds on Netlify.
+      # We can not rely on "Waiting for 200", since the user
+      # might push another commit to an existing PR, meaning
+      # the deploy preview is already online.
+      - name: Sleep 340 seconds
+        run: sleep 340
```

Comment:
Can be lowered once #1212 is merged

Reply:
Just merged #1212

Reply:
@BrunnerLivio

Reply:
@BrunnerLivio any ideas on what "sleep time" would be appropriate here :D ?

Reply:
Sorry for the late response - quite busy over here.
The build on Netlify itself is still around 6m - 8ms. Angular 9 is really not doing us a favor here hehe. So not much of an improvement since #1212. 
I am trying to get rid of this sleep command as a whole, when [JakePartusch/wait-for-netlify-action#6](https://github.com/JakePartusch/wait-for-netlify-action/issues/6) is resolved.

 I think as for now this is alright.

Worst case scenario is; user creates a PR -> Netlify deploys -> creates another commit starting a new deploy -> Gitlab CI did not wait long enough and uses the "old" deploy from the previous commit. 

What do you think?

Reply:
Sounds OK I guess ;)